lib/CheckboxItem/CheckboxItemAccessibilitySupport.js

```diff
@@ -0,0 +1,30 @@
+var
+	kind = require('enyo/kind');
+
+/**
+* @name CheckboxItemAccessibilityMixin
+* @mixin
+*/
+module.exports = {
+
+	/**
+	* @private
+	*/
+	observers: [
+		{method: 'updateAccessibilityAttributes', path: ['checked']}
+	],
+
+	/**
+	* @private
+	*/
+	updateAccessibilityAttributes: kind.inherit(function (sup) {
+		return function (props) {
+			var enabled = !this.accessibilityDisabled;
+			sup.apply(this, arguments);
+			this.setAttribute('role', enabled ? 'checkbox' : null);
+			this.setAttribute('tabindex', enabled ? 0 : null);
+			this.setAttribute('aria-checked', enabled ? String(this.checked) : null);
+			this.setAttribute('aria-labelledby', enabled ? this.getId() : null);
```

Comment:
Should this use the client's id rather than the control's id? Otherwise, it's declaring that it's labeling itself but it doesn't have any direct content.

```
this.setAttribute('aria-labelledby', enabled ? this.$.client.getId() : null);
```